### ISSUE TITLE
invocations from different py files with same function name cause cache colision

### ISSUE THREAD

**Issue by tenox7** [sarenehan/cache_to_disk#3]
```
foo.py:
@cache_to_disk(1)
def myfun():
```
will store the cache in exact same file as
```
bar.py:
@cache_to_disk(1)
def myfun():
```

causing a cache colision

**Comment by gerardsimons** [comment]
Yeah it seems it just takes as ID the name of the functions and the string representations of the arguments. It should also take the name of the module / script. The package is kind of a mess anyway as the string representation are awful when you have an object, as the pointer just changes every time.

**Comment by antoine79** [comment]
I've submitted [Pull Request #12](https://github.com/sarenehan/cache_to_disk/pull/12) for this issue
